src/wallet/rpcdump.cpp

```diff
@@ -243,6 +245,71 @@ UniValue importaddress(const UniValue& params, bool fHelp)
     return NullUniValue;
 }
 
+UniValue importprunedfunds(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() < 2 || params.size() > 3)
+        throw runtime_error(
+            "importprunedfunds\n"
+            "\nImports funds without rescan. Corresponding address or script must previously be included in wallet. Aimed towards pruned wallets.\n"
+            "\nArguments:\n"
+            "1. \"rawtransaction\" (string, hex, required) A raw transaction funding an already-existing address in wallet\n"
```

Comment:
nit: `(string, required)` would be the pattern that matches with other rpc commands help. Hex as string type is mostly mentioned in the parameters descriptions.